src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -46,7 +46,7 @@
 //** By default, this inherits from the `<body>`.
 @headings-font-family:    @font-family-base;
 @headings-font-weight:    400;
-@headings-line-height:    1.1;
+@headings-line-height:    1.3;
```

Comment:
I believe it's because bootstrap uses the line-height for certain calculations (like how tall this or that should be) but I'm not sure. It's bs (bootstrap).